src/modpacks/migrations/0002_auto_20160914_0050.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
Yes; newer versions of Django use migrations instead of South and `manage.py syncdb`.

When you make a change to the models, you're supposed to do `manage.py makemigrations`, commit the migrations along with the model changes, and then when you deploy the changes you do `manage.py migrate`.

Don't hate the player, hate the game. ;)